src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -170,15 +158,15 @@ function Overview({type, tenantName, className}: OverviewProps) {
             [EPathType.EPathTypePersQueueGroup]: () => <TopicInfo data={schemaData} />,
         };
 
-        return (
-            (type && pathTypeToComponent[type]?.()) || (
-                <SchemaInfoViewer fullPath={currentItem.Path} data={schemaData} />
-            )
-        );
+        return (type && pathTypeToComponent[type]?.()) || <TableInfo data={schemaData} />;
     };
 
     if ((loading && !wasLoaded) || (isEntityWithMergedImpl && !mergedChildrenPaths)) {
-        return renderLoader();
+        return <Loader size="m" />;
+    }
+
+    if (error) {
+        return <div className="error">{error.statusText}</div>;
```

Comment:
It would be better to use parsed error (I mentioned error parser in prev comment). Because as I remember it's a lot of error constructions in project.